documentation/terraform_example.md

```diff
@@ -0,0 +1,448 @@
+          + internal = 22
+          + ip       = "0.0.0.0"
+          + protocol = "tcp"
+        }
+    }
+
+  # docker_container.sshd[2] will be created
+  + resource "docker_container" "sshd" {
+      + attach           = false
+      + bridge           = (known after apply)
+      + container_logs   = (known after apply)
+      + exit_code        = (known after apply)
+      + gateway          = (known after apply)
+      + id               = (known after apply)
+      + image            = (known after apply)
+      + ip_address       = (known after apply)
+      + ip_prefix_length = (known after apply)
+      + log_driver       = "json-file"
+      + logs             = false
+      + must_run         = true
+      + name             = "docker_target_2"
+      + network_data     = (known after apply)
+      + restart          = "no"
+      + rm               = false
+      + start            = true
+
+      + ports {
+          + external = 2202
+          + internal = 22
+          + ip       = "0.0.0.0"
+          + protocol = "tcp"
+        }
+    }
+
+  # docker_image.sshd will be created
+  + resource "docker_image" "sshd" {
+      + id     = (known after apply)
+      + latest = (known after apply)
+      + name   = "rastasheep/ubuntu-sshd"
+    }
+
+Plan: 4 to add, 0 to change, 0 to destroy.
+
+Do you want to perform these actions?
+  Terraform will perform the actions described above.
+  Only 'yes' will be accepted to approve.
+
+  Enter a value: yes
+
+docker_image.sshd: Creating...
+docker_image.sshd: Creation complete after 6s [id=sha256:49533628fb371c9f1952c06cedf912c78a81fbe3914901334673c369376e077erastasheep/ubuntu-sshd]
+docker_container.sshd[2]: Creating...
+docker_container.sshd[0]: Creating...
+docker_container.sshd[1]: Creating...
+docker_container.sshd[2]: Creation complete after 1s [id=268e5d2572d7610f88d977b8bd37896830e6aa4777b0c7aded5e29b5fb8b0fb2]
+docker_container.sshd[1]: Creation complete after 1s [id=3e32b6d85cfdfaa823bfd3148aa0c9772ae1d896e993508d44088927406b95a5]
+docker_container.sshd[0]: Creation complete after 1s [id=f04670444f710a646d7994a9cb5a60d5b0ef42ce915df9e713f324340bb1dbfa]
+
+Apply complete! Resources: 4 added, 0 changed, 0 destroyed.
+```
+Check that containers are running
+```
+$ docker ps
+CONTAINER ID        IMAGE               COMMAND               CREATED              STATUS              PORTS                  NAMES
+3e32b6d85cfd        49533628fb37        "/usr/sbin/sshd -D"   About a minute ago   Up About a minute   0.0.0.0:2201->22/tcp   docker_target_1
+f04670444f71        49533628fb37        "/usr/sbin/sshd -D"   About a minute ago   Up About a minute   0.0.0.0:2200->22/tcp   docker_target_0
+268e5d2572d7        49533628fb37        "/usr/sbin/sshd -D"   About a minute ago   Up About a minute   0.0.0.0:2202->22/tcp   docker_target_2
+```
+
+### Now try connecting with bolt
+First test a connection with docker transport
+```
+$ bolt command run hostname -t 3e32b6d85cfd --transport docker
```

Comment:
inventory 1 output... I think we should start with a "static" inventory v2. that will help build the case for how nice the plugin is.